examples/actix-web/src/main.rs

```diff
@@ -56,3 +56,28 @@ async fn main() -> Result<()> {
     future::try_join(http, worker).await?;
     Ok(())
 }
+
+// #[cfg(test)]
+// mod tests {
+
+//     async fn push_email_test() -> () {
```

Comment:
This is possibly not the way to go. We care about the trait and this would only be additional but its not even run in CI